src/features/common/components/ButtonC/ButtonC.tsx

```diff
@@ -0,0 +1,15 @@
+import Button  from "react-bootstrap/Button";
+import styles from './ButtonC.module.scss';
+import { ReactNode } from "react";
+
+interface Props {
+    children: ReactNode;
+    onClick?: () => void;
+    additionalClasses?: string
+}
+
+const ButtonC: React.FC<Props> = ({children, onClick, additionalClasses}) => {
```

Comment:
I believe you added that 'C' at the end of Button because we import 'Button' from bootstrap lib. Bt it looks a bit confusing, so it's better to call your component 'Button' (cause that's a Button you gonna use all across the app and the one from bootstrap will be imported only here) and import the Buttton from bootstrap as 'import Button as ReactButtom from ...'